Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201305.java

```diff
@@ -122,7 +96,7 @@ public static String extractBirthdate(PRPAMT201306UV02ParameterList params) {
             if (CollectionUtils.isNotEmpty(birthTime.getValue())
                 && birthTime.getValue().get(0) != null) {
                 IVLTSExplicit birthday = birthTime.getValue().get(0);
-                LOG.info("Found birthTime in query parameters = " + birthday.getValue());
+                LOG.info("Found birthTime in query parameters = ", birthday.getValue());
```

Comment:
Aren't you missing the placeholders, "{ }", for you your parameterized logging?

Reply:
Fixed.